### ISSUE TITLE
Multiple Phone Numbers

### ISSUE THREAD

**Issue by completespec** [raventools/schema-creator#147]
I need to publish a company profile with two phone numbers, but the plugin only has a space to enter either one phone number and one fax number. Is there any way to add additional phone numbers?

**Comment by stephenyeargin** [comment]
I don't believe it currently supports this, unfortunately. The [specification](https://schema.org/faxNumber) has a designation for `faxNumber`, so that would be the ideal option.

**Comment by completespec** [comment]
Can I prevent the word "Fax" from showing before the second number? And, will search engines pick this up as a fax number instead of a second number?
